controller/system/logs.rb

```diff
@@ -48,7 +48,7 @@ class OnBoard::Controller
 
   # by path ## url_encode-ing is suggested to not confuse navbar
   # example: /system/logs/%2Fvar%2Flog%2Fmessages.html
-  get %r{^/system/logs/(.*)\.([\w\d]+)$} do
+  get %r{} do
```

Comment:
@ventre , this may resolve the issue on a particular device but I think we're telling Sinatra to execute the below code *always*, which is not what we want...

The logs functionality is broken and that's known, but this won't fix it...

Reply:
Particularly, here we're updating to Sinatra v2 which may be incompatible.... but it's "my fault" as the Gemfile (not Gemfile.lock) was too vague and did not specify a preferred version...